eks-anywhere-common/Addons/Partner/Nirmata/Kyverno/external-secret.yaml

```diff
@@ -14,3 +14,4 @@ spec:
   - secretKey: nirmata-license-key # which key it's going to be stored
     remoteRef:
       key: nirmata-license-key # Our secret-name goes here
+      property: nirmata-license-key
```

Comment:
Without it, the secret was taken as: {"nirmata-license-key":"free-tier-license"}